src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -34,10 +34,10 @@ const chartData = {
 
 export const MousePopulationsModel = types
   .model("MousePopulations", {
-    "initialEnvironment": "white" as EnvironmentColor,
-    "numHawks": 2,
-    "initialPopulation.white": 33.3,
-    "initialPopulation.tan": 33.3,
+    "initialEnvironment": types.string,
```

Comment:
This is done. I wasn't able to find a nice pattern to avoid the redundancy of

```
export type EnvironmentColors = "white" | "neutral" | "brown";

export const MousePopulationsModel = types
  .model("MousePopulations", {
    "environment": types.enumeration("environment", ["white", "neutral", "brown"]),
     ...
````

There *is* a way to do this better if we use `enum`, instead of sting literal unions. We could switch to that instead, but I'm still not sold on them as I find them uglier. Happy to go either way, though.

Reply:
Cancel that thought. I see you already have the `SpaceTypeEnum` and `SpaceType = SpaceTypeEnum.Type` pattern in @mklewandowski 's PR, so may as well continue the pattern here.

It seems a tiny bit backwards to me that the actual type is based on the MST type, rather than the other way around, but the other way to do it is apparently `color: types.enumeration<Color>("Color", Object.values(Color))`, which requires using `es2017` to get `Object.values`, and that doesn't seem worth it for this.

Anyway, switched to enum.